pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -313,6 +313,8 @@ def phase_str(phi: float) -> str:
         ch_data = data[ch]
         basis = ch_obj.basis
         ys = ch_data.get_input_curves()
+        ys_mod = [()] * 3
+        yseff = [()] * 3
```

Comment:
Before they were only declared when relevant. Now I "initialise" them empty so that I don't have to check for their existence when defining the ranges.